notebooks/calculate_sensitivity_from_Crab.ipynb

```diff
@@ -0,0 +1,2032 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # SECOND PART OF THE NOTEBOOK!!! RUN FROM HERE OF YOU JUST WANT TO RE-RUN THE CUT OPTIMIZATION  #
</pre><p>typo: OF -&gt; IF</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>